evalml/pipelines/components/transformers/imputers/per_column_imputer.py

```diff
@@ -18,6 +18,10 @@ class PerColumnImputer(Transformer):
         default_impute_strategy (str): Impute strategy to fall back on when none is provided for a certain column.
             Valid values include "mean", "median", "most_frequent", "constant" for numerical data,
             and "most_frequent", "constant" for object data types. Defaults to "most_frequent".
+        impute_all (bool): Whether or not to impute all columns or just the columns that are specified in `impute_strategies`. If True,
+            all columns will be imputed either using the strategy specified in `impute_strategies` or using the `default_impute_strategy`.
```

Comment:
I am not sure what you mean by "If True, all columns will be imputed either using the strategy specified in `impute_strategies`".  Aren't the `impute_strategies` defined on a per-column basis?

Reply:
@chukarsten Yup, `impute_strategies` is a dictionary that looks like `{ "col_name": {"impute_strategy": "mean", "fill_value": 0}, "col_name2": {"impute_strategy": "mean", "fill_value": 0} ... }`, but if they're not specified in the dictionary then we use the `default_impute_strategy` for that column. Does that make sense / do you think this could be clarified more?

Reply:
Updated the wording a bit!